src/share/poudriere/image.sh

```diff
@@ -155,14 +160,19 @@ excludelist=$(mktemp -t excludelist)
 mkdir -p ${WRKDIR}/world
 mkdir -p ${WRKDIR}/out
 [ -z "${EXCLUDELIST}" ] || cat ${EXCLUDELIST} > ${excludelist}
+# Some poudriere jail directory used for upgrading source or binary based jail
+# are useless on the final image
 cat >> ${excludelist} << EOF
 usr/src
+var/db/freebsd-update
+var/db/etcupdate
+boot/kernel.old
 EOF
 case "${MEDIATYPE}" in
 usb|*firmware|rawdisk)
 	truncate -s ${IMAGESIZE} ${WRKDIR}/raw.img
 	md=$(/sbin/mdconfig ${WRKDIR}/raw.img)
-	newfs -j -L ${IMAGENAME} /dev/${md}
+	newfs -L ${IMAGENAME} /dev/${md}
```

Comment:
We should find a way to make this change optional so one can decide if he wants journalising or not